shot/src/main/scala/com/karumi/shot/ShotPlugin.scala

```diff
@@ -72,16 +72,19 @@ class ShotPlugin extends Plugin[Project] {
       project.getExtensions.getByType[ShotExtension](classOf[ShotExtension])
     val instrumentationTask = extension.getOptionInstrumentationTestTask
     val packageTask = extension.getOptionPackageTestApkTask
-    (instrumentationTask, packageTask) match {
-      case (Some(instTask), Some(packTask)) =>
-        executeScreenshot.dependsOn(instTask)
-        pullScreenshots.dependsOn(packTask)
-      case _ =>
-        executeScreenshot.dependsOn(Config.defaultInstrumentationTestTask)
-        pullScreenshots.dependsOn(Config.defaultPackageTestApkTask)
-    }
 
-    executeScreenshot.dependsOn(DownloadScreenshotsTask.name)
+    if (extension.runInstrumentation) {
```

Comment:
It's fully possible I misunderstand this here but should the `executeScreenshot` task not depend on the `RemoveScreenshotsTask` (line 70) task unless `runInstrumentation` is `true`? 

Since we're not pulling the screenshots from a device when the instrumentation is not run, we're likely in an environment where neither a device nor adb is available (which is the case in the original issue).